e2e/tests/image-list.test.js

```diff
@@ -124,3 +184,21 @@ Scenario('Image list with hotkey navigation', async ({ I, LabelStudio, AtImageVi
   I.see('1 of 4');
 });
 
+Scenario('Image list exports correct data', async ({ I, LabelStudio, AtImageView }) => {
+  const params = {
+    config,
+    data,
+    annotations: [{ id: 1, result }],
+  };
+
+  I.amOnPage('/');
+  I.executeScript(initLabelStudio, params);
+
+  await AtImageView.waitForImage();
+  await AtImageView.lookForStage();
+
+  I.pressKey('Ctrl+d');
```

Comment:
Not sure what we are testing here? If this was to test duplication, wouldn't it require a region to be selected first?